Corgibytes.Freshli.Lib/GitFileHistoryFinder.cs

```diff
@@ -117,10 +117,13 @@ string pattern
         )
         {
             string clonedProjectRoot = NormalizeLocation(projectRootPath);
-            return Directory.GetFiles(clonedProjectRoot,
-                                      pattern,
-                                      SearchOption.AllDirectories)
-                            .Select(f => Path.GetFileName(f)).ToArray();
+            var files = Directory.GetFiles(clonedProjectRoot,
```

Comment:
If we aren't doing anything with `files` it could just be returned:

```csharp
return Directory.GetFiles(clonedProjectRoot, ....);
```

Reply:
Agreed